### PR TITLE
SITL: Add tiltquad vectored H-Frame

### PR DESCRIPTION
**Pull request by mirkix**
SITL: Add tiltquad vectored H-Frame and default parameters

### CODE REVIEW

File: Tools/autotest/pysim/vehicleinfo.py [hunk 1]
```diff
@@ -120,6 +120,11 @@ def __init__(self):
                 "waf_target": "bin/arduplane",
                 "default_params_filename": "default_params/quadplane-tilttrivec.parm",
             },
+                "quadplane-tilthvec": {
```
Comment (OXINARF) [inline Tools/autotest/pysim/vehicleinfo.py]: Nitpicking: wrong indentation

**Comment by mirkix** [comment]
@OXINARF fixed

File: Tools/autotest/pysim/vehicleinfo.py [hunk 2]
```diff
@@ -120,6 +120,11 @@ def __init__(self):
                 "waf_target": "bin/arduplane",
                 "default_params_filename": "default_params/quadplane-tilttrivec.parm",
             },
+            "quadplane-tilthvec": {
+                "make_target": "sitl",
+                "waf_target": "bin/arduplane",
+                "default_params_filename": "default_params/quadplane-tilthvec.parm",
```
Comment (peterbarker) [inline Tools/autotest/pysim/vehicleinfo.py]: Please consider inheriting the bulk of the parameters from `plane.parm`; you can see how Copter does a lot of that.

**Comment by mirkix** [comment]
@tridge I will update the parameter file, but it is based on https://github.com/ArduPilot/ardupilot/blob/master/Tools/autotest/default_params/quadplane-tilttrivec.parm with only some small changes.

**Comment by mirkix** [comment]
@tridge @peterbarker Done.

**Comment by tridge** [comment]
looks good, thanks!

**Comment by tridge** [comment]
btw, have you built one of these? got any photos/videos or logs?

**Comment by mirkix** [comment]
@tridge Not yet, but it should be my next project.

**Pull request merged**